.github/workflows/CMake.yml

```diff
@@ -9,6 +9,8 @@ on:
     branches: [ "main" ]
     paths-ignore:
       - 'rust/**'
+  schedule:
+    - cron: '17 17 * * 2'
```

Comment:
Why do we need a schedule/cron in addition to the push and pull_request triggers?

Reply:
That's what the rules said to do. Even if there's no changes, run it once a week.